code/modules/mob/living/carbon/human/species/station/traits_vr/positive.dm

```diff
@@ -62,6 +62,12 @@
 	cost = 2
 	var_changes = list("unarmed_types" = list(/datum/unarmed_attack/stomp, /datum/unarmed_attack/kick, /datum/unarmed_attack/claws, /datum/unarmed_attack/bite/sharp, /datum/unarmed_attack/bite/sharp/numbing))
 
+/datum/trait/fangs
+	name = "Numbing Fangs"
+	desc = "Provides fangs that makes the person bit unable to feel their body or pain."
+	cost = 1
+	var_changes = list("unarmed_types" = list(/datum/unarmed_attack/stomp, /datum/unarmed_attack/kick, /datum/unarmed_attack/claws, /datum/unarmed_attack/bite/sharp/numbing))
```

Comment:
`datum/unarmed_attack/claws` should probably be swapped for `/datum/unarmed_attack/punch` here, otherwise you've kinda-sorta given them both for a discount and only removed the non-injecting bite attack.

Reply:
Yeah, my bad, thanks for catching that